README.md

```diff
@@ -1,29 +1,63 @@
 # Global Forester API Example Scripts
 
-This repository contains a collection of scripts intended as a starting point for developers looking to utilize the Global Forester API using Swagger. These scripts can be used or combined as-is, or serve as inspiration for more complex implementations. To set up and run the scripts, follow the steps below.
+This repository contains a collection of Python scripts intended as a starting point for developers looking to utilize the Global Forester API using Python and Swagger. These scripts can be used or combined as-is, or serve as inspiration for more complex implementations. To set up and run the scripts, follow the steps below.
 
-## Getting Started
+See https://api.globalforester.com/swagger for more detailed information about the API.
 
-### 1. Generate a Swagger
+## Getting Started
 
-1. If you use homebrew, install [swagger-codegen](https://github.com/swagger-api/swagger-codegen/tree/3.0.0) with: `brew install swagger-codegen` (other installation methods are detailed [here](https://github.com/swagger-api/swagger-codegen))
-2. cd into _GlobalForester-API_
-3. Generate a swagger with: `swagger-codegen generate -i https://api.globalforester.com/swagger/v1/swagger.json -o Swagger -l python`
+### 1. Clone the repo
+```shell
+git clone https://github.com/peroper/GlobalForester-API.git
+cd GlobalForester-API
+```
 
-> Scripts assume that the swagger is located in a folder called 'Swagger'
+### 2. Install Python 3
+https://www.python.org
 
-### 2. Install Dependencies
+### 3. Install Python Dependencies
 
 It is recommended to install dependencies in a virtual environment. This will isolate project dependencies from other projects on the same computer. If you want to read more about virtual environments you can do so here: https://docs.python.org/3/library/venv.html
 
-1. Create a virtual environment with: `python3 -m venv venv`
-2. Activate the virtual environment with: `source venv/bin/activate`
-3. Install dependencies with `pip3 install -r requirements.txt`
+* Create a virtual environment
+```shell
+python3 -m venv venv
+```
+
+* Activate the virtual environment
+```shell
+source venv/bin/activate
+```
+
+* Install dependencies
+```shell
+pip3 install -r requirements.txt
+```
```

Comment:
Format these as a numbered list?

1. Create a virtual environment
```shell
python3 -m venv venv
```

2. Activate the virtual environment
```shell
source venv/bin/activate
```

3. Install dependencies
```shell
pip3 install -r requirements.txt
```